sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -115,6 +116,15 @@ case class CreateTableLikeCommand(
       CatalogTableType.EXTERNAL
     }
 
+    val newProperties = sourceTableDesc.tableType match {
+      case VIEW =>
+        // For view, we just use new properties
+        properties
```

Comment:
Keep view behavior as before. Hive also does not copy view properties.

Reply:
nit format;
```

    val newProperties = sourceTableDesc.tableType match {
      case MANAGED | EXTERNAL =>
        // Hive only retain the useful properties through serde class annotation.
        // For better compatible with Hive, we remove the metastore properties.
        sourceTableDesc.properties -- DDLUtils.METASTORE_GENERATED_PROPERTIES ++ properties

      case VIEW =>
        // For view, we just use new properties
        properties
    }
```